.github/workflows/master-build.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and merge-publish
```

Comment:
Curious about the difference between `master-build` and `pr-build`?
As pr-build is also getting run on trigger -  on:push, right?

The only difference I see here is that dockerPushImages. can we have `if condition` in pr-build that it happens only for merged to main. So, we don't need this entire workflow which is same.

Reply:
That is how it was earlier

Reply:
reverting it back to earlier state. We discussed having separate merge-publish so this was done.